### ISSUE TITLE
Description attribute in record elements

### ISSUE THREAD

**Issue by Iskander1986** [cogu/autosar#38]
I found a mismatch behavior, for tooling convention like a System Desk from DSpace it's possible to create a description attribute inside Record elements.
So to solve those issue i suggest to add between line 84 and 85 of file datatype_writer.py the following code:

```
tmp = self.writeDescXML(childElem)
if tmp is not None: lines.extend(self.indent(tmp,3))
```
This is the same code are used in other classes.

**Comment by cogu** [comment]
This is currently not prioritized and will be fixed later. If you need the fix faster you are welcome to create a pull-request.

**Comment by cogu** [comment]
* This won't be fixed in version 0.4.x.
* In version 0.5.x the desc element is supported equally well for all elements due to its new class design.
